packages/web-components/package.json

```
@@ -181,6 +181,7 @@
     "compile": "node ./scripts/compile",
     "compile:benchmark": "rollup -c rollup.bench.js",
     "clean": "node ./scripts/clean dist",
+    "exports:check": "node --no-warnings=ExperimentalWarning ./scripts/check-export-paths.js",
```

based on other comments

```suggestion
    "exports:check": "node ./scripts/check-export-paths.js",
```